src/sonic-yang-mgmt/setup.py

```diff
@@ -52,15 +28,26 @@ def run (self):
         'Programming Language :: Python :: 3.8',
     ],
     description="Package contains Python Library for YANG for sonic.",
-    tests_require = test_requirements,
     license="GNU General Public License v3",
     long_description=readme + '\n\n',
+    install_requires = [
+        'xmltodict==0.12.0',
+        'ijson==2.6.1'
+    ],
+    tests_require = [
+        'pytest==3',
+        'xmltodict==0.12.0',
```

Comment:
Is this necessary? It looks like xmltodict is only imported by the source, but not the tests.

Reply:
Test needs xmltodict, without it import fails, while testing. BTW test will mostly need all packages which dev needs,because in the test code we are calling the Dev code.